iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerGrammar.java

```diff
@@ -276,6 +281,61 @@ public UserInstruction USER() {
     );
   }
 
+  public Argument ARGUMENT() {
+    return b.<Argument>nonterminal(DockerLexicalGrammar.ARGUMENT).is(
+      f.argument2(
```

Comment:
What about argument method number 1? Do we still need it? Do we plan to remove it?